README.md

```diff
@@ -2,25 +2,21 @@
 
 __Copyright (c) 2007 - 2021 Unity Project by Mike Karlesky, Mark VanderVoord, and Greg Williams__
 
-Welcome to the Unity Test Project, one of the main projects of ThrowTheSwitch.org.
-Unity Test is a unit testing framework built for C, with a focus on working with embedded toolchains.
+Welcome to Unity Test Project, one of the main projects of ThrowTheSwitch.org.
+Unity Test is a unit testing framework built for C, which focuses on working with embedded toolchains.
```

Comment:
Both changes looks bad to me. Can you explain why they are needed?

Reply:
In fact, per [common grammar rules](http://www.kentlaw.edu/academics/lrw/grinker/LwtaThat_Versus_Which.htm), _that_ without a comma is more correct than _which_ with a common. I think the existing _with_ is quite sufficient (though it does not need a comma in front of it).